test/db/archos/linux-x64/dbg_dts

```diff
@@ -20,3 +20,28 @@ EXPECT=<<EOF
 0x00400565
 EOF
 RUN
+
+NAME=stop trace session (dts-)
+FILE=bins/elf/analysis/calls_x64
+ARGS=-d -e dbg.bpsysign=true
+BROKEN=1
+CMDS=<<EOF
+dcu main
+dts+
+ds
+ds
+ds
+ds
+dts-
+ds
+ds
+dt
```

Comment:
`dt` is unrelated here, you will get the same result after removing `dts+` and `dts-` since it's a different type of tracing(in general one of these should be renamed to avoid confusion).

You could test `dts-` by attempting to step back(`dsb`) after stopping the session.

Reply:
Thanks for the reply.

Okay, that makes sense, but what should I `EXPECT` over here?  Running `dsb` after stopping the session returns  `Session has not started`, but apparently rz-test can't catch that

Reply:
Maybe it's in `EXPECT_ERR`? Either way, you can print PC and see that you didn't actually step back.

Reply:
Sweet! Thanks for the really quick reply!